model/eventModel.js

```diff
@@ -37,4 +54,14 @@ const eventSchema = Mongoose.Schema({
     }
```

Comment:
`required` and `default` are not needed both since if there is a default value it will always be set.

Reply:
Event if i update it to an empty field?

Reply:
It should be, just remember to use that `options` object when updating.